### PR TITLE
Fix display on non store pages

### PR DESCRIPTION
**Pull request by salesfelipe**
- ditto.

- Before:
Screen Shot 2021-04-12 at 5 00 41 PM

- After: 
Screen Shot 2021-04-12 at 5 02 02 PM

### CODE REVIEW

**Comment by igorbrasileiro** [comment]
This PR isn't a patch, right? Because you changed the keys of messages.

**Comment by salesfelipe** [comment]
> This PR isn't a patch, right? Because you changed the keys of messages.

It depends on your interpretation.  For me, since I had a unexpected behavior and I fixed...

**Comment by igorbrasileiro** [comment]
> It depends on your interpretation. For me, since I had a unexpected behavior and I fixed...

But you would fix and would cause another bug, you know?!

**Review by arthurepc — APPROVED** [review]
LGTM

File: react/TranslateTotalizer.js [hunk 1]
```diff
@@ -24,9 +24,34 @@ export const knownTotalizers = defineMessages({
...
+  Interest: {
+    id: 'totalizers.Interest',
+    defaultMessage: 'Interest',
+  },
+})
+
+const TranslateTotal = ({ intl, id, totalizer, nonStorePage }) => {
   if (totalizer && totalizer.id) id = totalizer.id
 
+  if(nonStorePage && id in nonStoreKnownTotalizers) return intl.formatMessage(nonStoreKnownTotalizers[id])
```
Comment (klzns) [inline react/TranslateTotalizer.js]:
```suggestion
  if (nonStorePage && id in nonStoreKnownTotalizers) return intl.formatMessage(nonStoreKnownTotalizers[id])
```

**Review by klzns — APPROVED** [review]

**Pull request merged**